### ISSUE TITLE
[1.7 RC3][macOS] Minimizing Decrediton or DEX window for a long time makes all the buttons/screens unusable

### ISSUE THREAD

**Issue by xsanchezz** [decred/decrediton#3658]
Minimizing the DEX window and returning makes all the buttons and the screen unusable. Requires a forced close.

<img width="1504" alt="Screen Shot 2022-01-12 at 10 01 46 PM" src="https://user-images.githubusercontent.com/4775731/149252623-24d723fc-b0eb-4c1e-a157-d4d543ccf71a.png">

**Comment by xsanchezz** [comment]
After I have some BTC available to trade in DEX, how I can place an order? I don't see it

<img width="1336" alt="Screen Shot 2022-01-12 at 10 37 31 PM" src="https://user-images.githubusercontent.com/4775731/149256006-a8d7d14c-d7f1-41ad-aa32-647fdcbd9f19.png">

**Comment by xsanchezz** [comment]
Okay, just found how to solve the last screenshot reported. You need to send more than 0.10 DCR to the DEX wallet for DCR. If you send exactly 0.10, the DEX will not pay the registration fee.

**Comment by xsanchezz** [comment]
> After I got some BTC available to trade in DEX, how I can place an order? I don't see it
> 
> <img alt="Screen Shot 2022-01-12 at 10 37 31 PM" width="1336" src="https://user-images.githubusercontent.com/4775731/149256006-a8d7d14c-d7f1-41ad-aa32-647fdcbd9f19.png">

Okay, just found how to solve the last screenshot reported. You need to send more than 0.10 DCR to the DEX wallet. If you send exactly 0.10, the DEX will not pay the registration fee.

**Comment by xsanchezz** [comment]
@alexlyp I have found a new behavior. I started the mixer, and minimized the window. I closed the lid of the computer, so, it goes to sleep. Next day, when I turn on the computer again and reopen Decrediton, the screen comes in blank.

<img width="1218" alt="Screen Shot 2022-01-20 at 3 08 41 PM" src="https://user-images.githubusercontent.com/4775731/150405754-8c937b7c-058d-4254-8a0e-59dc2865ba73.png">
